app/src/features/surveys/observations/observations-table/grid-column-definitions/GridColumnDefinitionsUtils.tsx

```diff
@@ -104,7 +104,13 @@ export const getEnvironmentColumnDefinitions = (
   for (const environment of environments.quantitative_environments) {
     colDefs.push(
       ObservationQuantitativeEnvironmentColDef({
-        environment: environment,
+        environment: {
+          ...environment,
+          name: environment.unit
+            ? `${environment.name} (${environment.unit}${environment.unit.endsWith('er') ? 's' : ''})`
```

Comment:
suggestion: We should place this logic inside its own function to use if statements instead of nested ternarys

ie: getQuantitativeEnvironmentHeaderName

Reply:
There is also a `pluralize` util function that might be of use

Reply:
I'd also move this modification code outside of the `colDefs.push(...)`

```
  for (const environment of environments.quantitative_environments) {
   const formattedEnvironment = <do stuff to modify the name, etc>

    colDefs.push(
      ObservationQuantitativeEnvironmentColDef({
        environment: formattedEnvironment,
        hasError: hasError
      })
    );
  }
  ```